lightkurve/periodogram.py

```diff
@@ -2,64 +2,251 @@
 from astropy import units as u
 from astropy.stats import LombScargle
 from matplotlib import pyplot as plt
+from scipy.ndimage.filters import gaussian_filter
+from astropy.units import cds
+from astropy.table import Table
+import astropy
 
-__all__ = ['Periodogram']
+import logging
+from . import PACKAGEDIR, MPLSTYLE
 
 
+log = logging.getLogger(__name__)
+
+__all__ = ['Periodogram', 'estimate_mass', 'estimate_radius',
+            'estimate_mean_density', 'stellar_params', 'standardize_units']
+
+###Add uncertainties <- review these [astero]
+numax_s = 3090.0 # Huber et al 2013
+deltanu_s = 135.1
+teff_s = 5777.0
+
+class InputError(Exception):
+    """Raised if user inputs both frequency and period kwargs"""
+    pass
+
 class Periodogram(object):
+    ###Update this [houseekeping]
     """Represents a power spectrum.
-
     Attributes
     ----------
+    lc : `LightCurve` object
+        The LightCurve from which the Periodogram is computed. Holds all  of
+        the `LightCurve object's properties.
     frequencies : array-like
         List of frequencies.
-    powers : array-like
-        Power measurements.
+    periods : array-like
+        List of periods (1 / frequency)
+    power : array-like
+        The power-spectral-density of the Fourier timeseries.
+    nyquist : float
+        The Nyquist frequency of the lightcurve.
```

Comment:
I recommend specifying the unit expected.  (Probably the same as the unit of the frequencies used as input?  If that's the case, let's say so.)

Reply:
Of the power, nyquist, or both? Either way, I'll make sure to specify their relation to the freq_unit parameter, which is defined just below.